config/locations.yml

```diff
@@ -103,165 +104,138 @@ covenantgn:
 covenantmh:   
   name: "Covenant Health - Misericordia Community Hospital"
   locationurl: "​https://www.covenanthealth.ca/corporate-information/library-services​"
-cross:   
-  name: "Alberta Health Services - Cross Cancer Institute"
-  locationurl: 
-cuc:   
-  name: "Burman University"
+cross:
+  name: "Cross Cancer Institute"
+  locationurl:
+cuc:
+  name: "Canadian University College"
```

Comment:
Not sure if this is correct.  The url is "Burman University".  Also duplicated in `burman`

Reply:
@pgwillia I was mistaken, this code is no longer used - it's not in the Sirsi libraries. The current/correct code is Burman